### DISCUSSION TITLE
BXNUC10i3FNK4  Installation failed

### DISCUSSION THREAD

**Discussion by digitaldiamont** [hackintosh-club/intel-nuc10#107] [Q&A]
Hi guys, it's driving me crazy.

I have 2 Intel NUC 10i3, but only one allows the installation of MacOS.
I would appreciate it if anyone has an idea for me.

1st computer: BXNUC10i3FNH2 Manufactured 2022

Installation without any problems, everything works.

2nd computer: BXNUC10i3FNK4 Manufactured 2020

Despite having the same BIOS (Version 55) and the same settings, I can't even boot into the installation process.
CFG value is set to 0 on both computers.

Even if I install the SSD from computer #1 into computer #2, MacOS still won't boot.

**Comment by apollohackintosh** [comment]
Can you boot with the same usb on the 1st computer?
